### DISCUSSION TITLE
Gradle wrapper task number of executions

### DISCUSSION THREAD

**Discussion by JavierSegoviaCordoba** [renovatebot/renovate#11598] [Request Help] [answered]
How many times is called the task `./gradlew wrapper --gradle-version=$version`? If the task is run only one time, it will only update the version in the properties file. To update the rest of the script files, it should be run two times in a row.

**Comment by rarkins** [comment, marked as answer]
If so, could you please:
- Create a bug report issue
- Provide a minimal reproduction repo which currently *fails* to update correctly
- Link to any documentation about this
